alpha_automl/builtin_primitives/image_encoder.py

```diff
@@ -32,7 +32,10 @@ def transform(self, images):
         data = []
         if isinstance(images, pd.DataFrame):
             for file in images[images.columns[0]]:
-                im = imread(file)
+                try:
+                    im = imread(file)
+                except:
+                    im = np.zeros([224, 224, 3], np.uint8)
```

Comment:
This (and also on lines 51-52) should be considered a severe error as training a model with such invalid files will compromise model quality. We need to notify the user with log messages so that they can take action, e.g., remove the broken files from the training data or fix them.

For now, I would just add log messages (ERROR level) here but maybe we should also discuss if there is a way to stop the pipeline from running to make sure the user is aware of the problem.

Reply:
Error message added